src/ytdl_sub/entries/soundcloud.py

```diff
@@ -105,6 +112,7 @@ def album_tracks(self, skip_premiere_tracks: bool = True) -> List[SoundcloudAlbu
                 album=self.title,
                 album_year=self.album_year,
                 soundcloud_track=track,
+                total_tracks=self.track_count,
```

Comment:
maybe we should incorporate this into the PlaylistMetadata as `playlist_count`. Then in `total_tracks`, return `self._playlist_metadata.playlist_count`

I think the ytdl field is 'playlist_count',  i recommend using a debugger and seeing what ytdl returns in the entry dict

Reply:
That would help solve this issue: https://github.com/jmbannon/ytdl-sub/issues/9

Reply:
Yea that makes sense, I've incorporated it.

Reply:
I've also launched it in a debugger and checked the entry dict, and the field is playlist_count, this is also used in the track_count property in  SoundcloudAlbum.

Reply:
Sweet looks great! Looks like the linter errored: https://github.com/jmbannon/ytdl-sub/runs/6236060213?check_suite_focus=true

One last nit: lets change the soundcloud variable name from `total_tracks` to `track_count`, that way it's near the other track variables via sorting in the documentation

Reply:
I believe I've fixed that linter error, and I've changed the variable name.